src/main/java/sdu/se9/tv2/management/system/persistence/PersistenceDatabaseHelper.java

```diff
@@ -30,12 +30,26 @@ public static void main(String[] args) {
         // Get connection
         Connection connection = getConnection();
 
-        // Make query
+        //Creating the tables in database of they do not already exist
         try {
-            PreparedStatement stmt = connection.prepareStatement("SELECT * FROM SomeTable");
-            ResultSet result = stmt.executeQuery();
+            PreparedStatement stmt = connection.prepareStatement("CREATE TABLE IF NOT EXISTS Person (" +
+                    "id serial PRIMARY KEY," +
+                    "first_name varchar(20) NOT NULL," +
+                    "last_name varchar(20) NOT NULL" +
```

Comment:
I følger ikke design modellen for databasen, det hedder firstName og lastName, ikke first_name og last_name.